cinema/serializers.py

```diff
@@ -68,13 +65,80 @@
             "movie_title",
             "cinema_hall_name",
             "cinema_hall_capacity",
+            "tickets_available",
         )
 
 
+class TicketMovieSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Ticket
+        fields = ["row", "seat"]
+
+
 class MovieSessionDetailSerializer(MovieSessionSerializer):
     movie = MovieListSerializer(many=False, read_only=True)
     cinema_hall = CinemaHallSerializer(many=False, read_only=True)
+    taken_places = TicketMovieSerializer(source="tickets", many=True, read_only=True)
```

Comment:
The `taken_places` field uses `TicketMovieSerializer` with the source `tickets`. Ensure that the `related_name` for the `tickets` field in the `Ticket` model is correctly set to match this source.